sync3/extensions/receipts_test.go

```diff
@@ -16,10 +16,14 @@ func TestLiveReceiptsAggregation(t *testing.T) {
 	ext := &ReceiptsRequest{
 		Core: Core{
 			Enabled: &boolTrue,
+			Lists:   []string{"*"},
+			Rooms:   []string{"*"},
```

Comment:
Surely this is the default? Why are we needing to add this to tests?

Reply:
These are indeed the defaults when you send a request into the sync v3 logic (i.e. when it gets combined with the previous requests into muxedReq). This test does not do that: it is invoking `AccountDataRequest.AppendLive` directly. The typing test below is similar.